docs/tutorials/otiotool.md

```diff
@@ -0,0 +1,387 @@
+# otiotool Tutorial
+
+`otiotool` is a command-line utility in OpenTimelineIO for inspecting, manipulating, and transforming OTIO timeline files. This tutorial covers its main features and usage patterns, with practical examples.
+
+## Installation
+
+`otiotool` is included with several other command line utilities as part of the
+OpenTimelineIO Python module. You can install it via typical Python utilities
+like `pip`, etc. See [Quickstart](./quickstart]) for details.
```

Comment:
```suggestion
like `pip`, etc. See [Quickstart](./quickstart) for details.
```
I think this will fix the issue with the invalid link. If it doesn't, try `[Quickstart](quickstart)`.

Reply:
Oops. Thanks @JeanChristopheMorinPerso !